sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDB.scala

```diff
@@ -120,6 +120,9 @@ class RocksDB(
       if (conf.resetStatsOnLoad) {
         nativeStats.reset
       }
+      // reset resources to prevent side-effects from previous loaded version
```

Comment:
We did this in rollback case but not in commit case. This doesn't bring problems in current implementation of session window, but I think we should fix it to avoid future correctness problem.